### ISSUE TITLE
Angular 13 support

### ISSUE THREAD

**Issue by guyutongxue** [materiahq/ngx-monaco-editor#72]
While updating Angular to v13, I got these errors:
```
npm ERR! While resolving: my-project@1.0.0
npm ERR! Found: @angular/common@13.0.0
npm ERR! node_modules/@angular/common
npm ERR!   @angular/common@"~13.0.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @angular/common@"9.x || 10.x || 11.x || 12.x" from @materia-ui/ngx-monaco-editor@5.1.0
npm ERR! node_modules/@materia-ui/ngx-monaco-editor
npm ERR!   @materia-ui/ngx-monaco-editor@"5.1.0" from the root project
```

Seems that a "13.x" peerDependencies is required.

Also, add support for RxJS 7 (defaulted in Angular 13) if available. Thanks!

**Comment by mikerentmeister** [comment]
@Guyutongxue - You can use the --force flag to get past the error for now. However, this repo will need to get updated to build with Ivy instead to not have to use ngcc

**Comment by jmarc-roy** [comment]
I published **v6.0.0-beta.1** for the moment, with partial ivy compilation mode for Angular v13 (to use with latest monaco-editor v0.30.1). 
I wait for Stackblitz Angular v13 support before publishing stable new version.

**Comment by ericdesa** [comment]
Hi, just to let you know that Angular v13 seems to be available on stackblitz
- Stackblitz ticket : https://github.com/stackblitz/core/issues/1657
- Material examples running on v13 : https://material.angular.io/components/button/examples#button-overview

But i didn't find the **ngx-monaco-editor v6.0.0-beta.1**, so i don't know if i can help testing it.

**Comment by jmarc-roy** [comment]
Hello @ericdesa,

Thanks for the insights.
Will try to release v6.0.0 this week, quite busy this last weeks.

In the meantime, here is a live example with angular v13 and v6.0.0-beta.1: [Stackblitz](https://stackblitz.com/edit/materia-ngx-monaco-editor-example-cka1m6?file=src%2Fapp%2Fapp.module.ts).

**Comment by jmarc-roy** [comment]
Angular v13 support has been released with **v6.0.0**.
